lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_FallingSnowLevel.py

```diff
@@ -131,55 +131,154 @@ def test_no_fill_if_conditions_not_met(self):
         self.assertArrayEqual(self.snow_level_data, expected)
 
 
+class Test_linear_wet_bulb_fit(IrisTest):
+
+    """Test the linear_wet_bulb_fit method."""
+
+    def setUp(self):
+        """ Set up arrays for testing."""
+        # Set up a wet bulb temperature array with a linear trend near sea
+        # level. Some lines will cross the x-axis above zero and some below.
```

Comment:
Could we define what x is rather than just saying x; this goes for other places in this code as well. I think it would all be a little clearer if x had some physical meaning rather than being spoken of in the abstract.

Reply:
Done